resolwe_bio/processes/expression/cuffnorm.yml

```diff
@@ -103,21 +126,25 @@
     program: |
       experiment='{% for experiment in cuffquant %}{{ experiment.cxb.file }},{% endfor %}'
       replicates='{% for r in replicates %}{{r}},{% endfor %}'
-      labels='{% for l in labels %}{{l}},{% endfor %}'
+      sample_names='{% for cq in cuffquant %}{{cq|sample_name}},{% endfor %}'
 
-      labels=${labels%?}
-      experiment=${experiment%?}
+      experiment=${experiment%?} #removes comma at the end
       replicates=${replicates%?}
+      sample_names=${sample_names%?}
+     
+      labels=`cuffnorm_group_labels.py ${replicates}` #construction of group labels
```

Comment:
You can generate labels in the same way as replicates, so you don't need an extra script:

    labels='{% for r in replicates %}Group{{ r }},{% endfor %}'